otsdaq/FECore/FEVInterface.cc

```diff
@@ -133,12 +135,21 @@ void FEVInterface::addSlowControlsChannels(ConfigurationTree
 						getInterfaceUID() << "/" << groupLinkChild.first
 		            << "\t Type:" << groupLinkChild.second.getNode("ChannelDataType") << __E__;
 
+		// Unit transforms
+		std::string transformation = "";
+		try {
+			transformation = groupLinkChild.second.getNode("Transformation").getValue<std::string>();
+		} catch (...) {
+			__FE_COUT__ << "Slow controls 'Transformation' setting found." << __E__;
```

Comment:
Should this be "setting not found" instead of "setting found"?

Reply:
Thanks for spotting that, it's fixed now.